include/fmt/core.h

```diff
@@ -238,6 +238,7 @@
 
 #ifndef FMT_MODULE_EXPORT
 #  define FMT_MODULE_EXPORT
+#  define FMT_MODULE_EXPORT_CONSTANT static
```

Comment:
It seems that the name `FMT_MODULE_EXPORT_STATIC` would be more appropriate.

Reply:
Well, this depends on the point of view:
 - constants exported from module: external linkage required
 - constants in header of a traditional lib: internal linkage recommended

These macros have 'module export' in their name, therefore the choice how to annotate constants in the API surface.

Reply:
> constants exported from module: external linkage required

What error do you get without external linkage? Would replacing static with inline solve the problem?

Reply:
Nothing with internal or no linkage can be exported from any sort of module.
`static` -> `inline` at namespace scope is fine because this changes linkage from internal (with possibly multiple instances in the resulting program) to external (with just one instance in the program).

Reply:
Then I suggest using `inline` instead of making this module-dependent. `inline` seems more appropriate anyway.

Reply:
Yeah, but this introduces the problem that `inline` variables weren't a thing before C++17.

I suggest to just go with 'naked' `constexpr` variables like f.e. `invalid_arg_index` in `core.h`:2651. This is the current state of the updated PR.

Reply:
Sounds good.